miden/tests/integration/operations/field_ops.rs

```diff
@@ -661,7 +684,31 @@ proptest! {
         let test = build_op_test!(asm_op, &[base, pow]);
         test.expect_stack(&[expected.as_int()]);
         assert!(test.execute().is_ok());
+
+        //----------------------- exp with parameter containing pow ----------------
+
+        let build_asm_op = |param: u64| format!("exp.{}", param);
+        let base = a;
+        let pow = b;
+        let expected = Felt::new(base).exp(pow);
+
+        let test = build_op_test!(build_asm_op(pow), &[base]);
+        test.expect_stack(&[expected.as_int()]);
+        assert!(test.execute().is_ok());
```

Comment:
Do we need to test for `test.execute().is_ok()`? Isn't it implied by `test.expect_stack()`?